mirrormanager2/lib/hostconfig.py

```diff
@@ -25,9 +25,14 @@
 
 import mirrormanager2.lib
 
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
Ok cool :)

Does that work fine as well when specifying the logging via a config file?